src/sharedHooks/useFontScale.ts

```diff
@@ -1,15 +1,20 @@
 import {
+  useEffect,
```

Comment:
The changes to this file (`useFontScale.ts`) look unrelated to the PR's primary purpose, but I don't think they're harmful.

Reply:
This was mostly to remove some effects that were running after tests were finishing, which triggers warnings. If I'm touching tests I generally try to make them pass as cleanly as possible. The only risk I could see was that this hook would run less frequently and the app would not respond immediately to changes in system font size, but as far as I can tell it still does.